.github/workflows/yaml-lint.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: yamllint
+"on":
+  pull_request:
+    branches: [main]
+jobs:
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Lint YAML files
+        run: yamllint .
```

Comment:
> create a GitHub Action that installs yamllint and then runs it

It looks like it doesn't even install it. Is yamllint just pre-installed on GH runners?

Reply:
Yes, it is

Reply:
Nice. Could you just update the commit message?